File: sources/CSharp/src/Ers/System/InterpreterRenderSystem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ers.Engine;

namespace Ers
{
    public static class InterpreterRenderSystem
    {
        public static void Render2D() { ErsEngine.ERS_InterpreterRenderSystem_Render2D(); }

        public static void BuildMesh3D(Mesh mesh) { ErsEngine.ERS_InterpreterRenderSystem_BuildMesh3D(mesh.Data); }
    }
}

File: sources/CPP/include/Ers/Systems/InterpreterRenderSystem.cpp
#include "InterpreterRenderSystem.h"
#include "Ers/Api.h"
#include "Ers/Visualization/Mesh.h"

namespace Ers::Systems
{
    void InterpreterRenderSystem::Render2D()
    {
        ersAPIFunctionPointers.ERS_InterpreterRenderSystem_Render2D();
    }

    void InterpreterRenderSystem::BuildMesh3D(Ers::Mesh& mesh)
    {
        ersAPIFunctionPointers.ERS_InterpreterRenderSystem_BuildMesh3D(mesh.Data());
    }
} // namespace Ers::Systems
